python/pyspark/sql/dataframe.py

```diff
@@ -1077,7 +1077,7 @@ def sort(self, *cols, **kwargs):
         """Returns a new :class:`DataFrame` sorted by the specified column(s).
 
         :param cols: list of :class:`Column` or column names to sort by.
-        :param ascending: boolean or list of boolean (default True).
+        :param ascending: boolean or list of boolean (default ``True``).
```

Comment:
What does the double-back-tick do, out of curiosity?

Reply:
It formats the True to show up as slightly different text that is for python code expressions